### PR TITLE
[DevTools] Disable sizeBot on DevTools Rull Request

### PR DESCRIPTION
**Pull request by ryota-murakami**
Because DevTools code doesn't affect production bundle size.
Meaningless sizeBot comment give us frastration within Pull Request discussion.

### CODE REVIEW

File: dangerfile.js [hunk 1]
```diff
@@ -113,6 +113,10 @@ function row(result) {
     return;
   }
 
+  // Disable sizeBot in a Devtools PR like bellow title. Because that doesn't affect production bundle size.
+  // [Fix DevTools console announcement being suppressed by Fast Refresh #21864](https://github.com/facebook/react/pull/21864)
+  if (new RegExp('devtools?', 'i').test(danger.github.pr.title)) return;
```
Comment (bvaughn) [inline dangerfile.js:118]: I like the idea of this pull request, but I think maybe we shouldn't rely on the name of the title. (There may be DevTools-focused PRs that also change things in non-DevTools code.)

If we can instead look at the list of changed files and skip it if all files are in `packages/react-devtools*` then that would be okay.

**Review by bvaughn — CHANGES_REQUESTED** [review]

File: dangerfile.js (hunk 1 above)
Comment (ryota-murakami) [reply dangerfile.js:118]: @bvaughn Thank you for your decent review!
Totally I agree with you, actually I could only achieved by this way in a short spare time 😅

> look at the list of changed files and skip it if all files are in packages/react-devtools* then that would be okay.

I'll follow and implement this safety way and then I'll mention you review request again.
To making **detect only `packages/react-devtools*` files** logic I wanna little bit explore codebase for seek something effective util code in this situation.
So might be I'll update my work within about a week.


**misc.** I searched about toplevel **return** statement in node.js and stackoverflow says that equivalent to `process.exit()`.
I learned `return` `process.exit(0)` `process.exit()` are all equivalent exit status today 🤗

Comment (bvaughn) [reply dangerfile.js:118]: > I'll follow and implement this safety way and then I'll mention you review request again.
To making **detect only `packages/react-devtools*` files** logic I wanna little bit explore codebase for seek something effective util code in this situation.
So might be I'll update my work within about a week.

Sounds good 👍🏼

**Review by crypt096 — APPROVED** [review]

**Comment by ryota-murakami** [comment]
@bvaughn I implemented logic to check wether Devtools package files only committed.  
How is that?

**Review by bvaughn — APPROVED** [review]
Nice

**Comment by ryota-murakami** [comment]
Thank you for your review!

**Pull request merged**